docker-compose/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.6
```

Comment:
Remove the `docker-compose` directory if it is no longer used.

Reply:
done